xclim/core/units.py

```diff
@@ -252,6 +252,7 @@ def str2pint(val: str) -> pint.Quantity:
         return units.Quantity(1, units2pint(val))
 
 
+# FIXME: The typing for this needs to be addressed. Maybe we need to use generics here to help specify what goes in/out?
```

Comment:
Not sure I understand the comment. Doesn't `Quantified` cover exactly what this function accepts ? `DataArray`, `str` and `pint.Quantity`.

Reply:
Good point. This comment should be placed elsewhere. `convert_units_to` is consistent/accurate.